### PR TITLE
src/main.cpp: CodeQuality

### PR DESCRIPTION
**Pull request by hb9fxq**
- CodeQuality: Code style fixes for main.cpp
- Project Setup: Added /CMakeLists.txt.user .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,4 @@
 /version.txt
 .idea/
 cmake-build*/
-
+/CMakeLists.txt.user
```
Comment (argilo) [inline .gitignore:10]: Could you put newlines at the end of these files? Thanks.

Comment (hb9fxq) [reply .gitignore]: Done

File: src/applications/gqrx/main.cpp [hunk 2]
```diff
@@ -231,4 +228,4 @@ static void list_conf(void)
             std::cout << conf_files.at(i).toLocal8Bit().constData() << std::endl;
         }
     }
-}
+}
```
Comment (argilo) [inline src/applications/gqrx/main.cpp:231]: A newline is needed here as well.

Comment (hb9fxq) [reply src/applications/gqrx/main.cpp]: Got it :-)

**Pull request merged**